src/pymola/tree.py

```diff
@@ -189,7 +189,7 @@ def walk(self, listener: TreeListener, tree: ast.Node) -> None:
         if hasattr(listener, 'enter' + name):
             getattr(listener, 'enter' + name)(tree)
         for child_name in tree.__dict__.keys():
-            if isinstance(tree, ast.Class) and child_name in ('parent') or \
+            if isinstance(tree, ast.Class) and child_name in 'parent' or \
```

Comment:
Instead of `in 'parent'` it should probably be `== 'parent'`. The reason for the `in ...` is because this was copied from the check in `ast.py` iirc, where it checks on `('parent', 'root')`.